tests/unit/src/Internal/Payment/State/InitialStateTest.php

```diff
@@ -38,6 +38,12 @@ class InitialStateTest extends WCPAY_UnitTestCase {
 	 * @var InitialState
 	 */
 	private $sut;
+	/**
+	 * Service under test.
+	 *
+	 * @var StateFactory|InitialState
```

Comment:
```suggestion
	 * @var MockObject|InitialState
```

Just curious, doesn't PHPStorm highlight errors because of this? I'm noticing that @jessy-p's tests have something similar.

Reply:
It should, but perhaps it doesn't check tests. I will check that.